scripts/test/DynamicField/EditFieldRender.t

```diff
@@ -79,18 +79,27 @@ my $SecondUserID = $UserObject->UserAdd(
 );
 ok( $SecondUserID, 'Creation of second agent' );
 
+my $LayoutObject = Kernel::Output::HTML::Layout->new(
+    Lang         => 'en',
+    UserTimeZone => 'UTC',
+);
+
 # collect user data and build string for html selection comparison
 my %UserList = $UserObject->UserSearch(
     Search => '*',
     Valid  => 1,
 );
 my %UserLookup = map {
-    $_ => $UserObject->UserName(
+    my $UserName = $UserObject->UserName(
         UserID => $_,
-    )
+    );
+    my %Preferences = $UserObject->GetPreferences(
+        UserID => $_
+    );
+    $_ => $LayoutObject->Ascii2Html( Text => qq{"$UserName" <$Preferences{UserEmail}>} )
 } keys %UserList;
 
-my $UserSelectionString               = '  <option value="">-</option>';
+my $UserSelectionString;
```

Comment:
This should be initialized with an empty string in ordert to avoid a warning later.

Reply:
Ah, hm... perhaps you're right. I encountered the problem that the join later on places a newline at the beginning, but I had to remove this anyways by regex, so I think empty string initialization is fine. I will change this, thank you!